### DISCUSSION TITLE
GSF Brighton

### DISCUSSION THREAD

**Discussion by anewmandev** [Green-Software-Foundation/discussions#46] [Green Software Meetups]
Hello. 

I'm Adam Newman, I'm going to be co-hosting GSF Brighton (UK) with Oliver Winks [ohuu](https://github.com/ohuu). 

I wanted to introduce myself ("hi") and outline what we're planning for our Brighton Meetup to see if anyone has any thoughts or input. 

Our first event is planned for Thurs 28th September (we will confirm this date next week). Oli and I will do a welcome talk: introduce ourselves, set out our thoughts for building a local community, an intro to the GSF and cover some core Green Software concepts. We'll have some drinks and snacks put on. Future meetups will cover a range of relevant subject matter and a mix of different formats; guest speakers, round tables, show and tells, lightnight talks etc. With the pub thrown in here and there :)  

Am keen to hear if other organisers have any input on the above or ideas of things that work well for them (and also not). All thoughts and feedback are welcome. 

Also keen to connect with any individuals or organisations locally who are interested in the Brighton meetup. You can message me here or email me adam@cluelight.com. 

Thanks all :) look forward to crossing paths with some of you in the future. 

Adam

**Comment by NAMRATA-WOKE** [comment]
Here's the link for anyone who wants to join the Brighton Meetup: https://www.meetup.com/gsf-brighton/
